### PR TITLE
Week3tasks

### PR DESCRIPTION
**Pull request by zainab-bhadmus**
Week 3 tasks solution.

### CODE REVIEW

File: week 3-4/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
```
Comment (Easybuoy) [inline week 3-4/index.html]: Can you make all your imports use a relative path?

e.g `"./css/style.css"`

File: week 3-4/index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <title>To-do App </title>
+  </head>
+  <body>
+   <div class="main-div">
+        <input type="text" id="input"
```
Comment (Easybuoy) [inline week 3-4/index.html]: You did not close this input tag, thus not making it possible for the Todo list to work

File: week 3-4/index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        <button id= "enterBtn">Enter</button>
+        <hr>
+        <ul>
+
+        </ul>
+    </div>
+    <script type="text/javascript" src="app.js"></script>
```
Comment (Easybuoy) [inline week 3-4/index.html:19]: You can make this use a relative tag as well

**Review by Easybuoy — COMMENTED** [review]
You were only able to do the add todo functionality, can you work on adding the delete functionality to the app.

**Review by Easybuoy — APPROVED** [review]
I think you need to improve your knowledge on DOM manipulations, I would encourage you to build this app or any other app again using DOM manipulations. You can check out videos on Youtube on DOM manipulations. 

Try to work on 
- Deleting a TODO
- Marking a TODO as done/completed.

**Comment by Easybuoy** [comment]
> I think you need to improve your knowledge on DOM manipulations, I would encourage you to build this app or any other app again using DOM manipulations. You can check out videos on Youtube on DOM manipulations.
> 
> Try to work on
> 
> * Deleting a TODO
> * Marking a TODO as done/completed.

@zainab-bhadmus

**Comment by zainab-bhadmus** [comment]
Feedback is well received. Would work on it.

**Pull request merged**